### PR TITLE
Up rollup version

### PR DESCRIPTION
**Pull request by LeXXik**
Fixes https://github.com/advisories/GHSA-gcx4-mw62-g8wm

I confirm I have read the [contributing guidelines](https://github.com/playcanvas/engine/blob/master/.github/CONTRIBUTING.md) and signed the [Contributor License Agreement](https://docs.google.com/a/playcanvas.com/forms/d/1Ih69zQfJG-QDLIEpHr6CsaAs6fPORNOVnMv5nuo0cjk/viewform).

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "playcanvas",
   "version": "2.1.0-dev",
-  "lockfileVersion": 3,
```
Comment (willeastcott) [inline package-lock.json:4]: Looks like you're on an old version of NPM. The `package-lock.json` has been bumped down to 2 from 3. This adds about 7.500 lines to the file!

Comment (LeXXik) [reply package-lock.json:4]: Ouch

Comment (LeXXik) [reply package-lock.json:4]: Hmm, updated npm to 10.8.3 and getting warnings now. Do you have the same? What version do you use?

image

Comment (willeastcott) [reply package-lock.json:4]: I'm using 9.7.1. But those warnings are fine. The `eslint-plugin-import` team are working to add support for ESLint 9. I think they'll be done in a week or two. Then, those warnings will go away.

**Comment by LeXXik** [comment]
Getting many tasks failing :/ Is it something I should fix on my side?

**Pull request closed without merging**